Italian/scappare/cercare-uscita.md

```diff
@@ -0,0 +1,6 @@
+Ti alzi per cercare una via d'uscita.
+Inciampi in una bottiglia rotta a metà, e ci sono vetri ovunque.
+
+[La raccogli](scappare/raccogli.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.